src/workerd/api/form-data.h

```diff
@@ -109,6 +109,19 @@ class FormData: public jsg::Object {
 
     JSG_METHOD(forEach);
     JSG_ITERABLE(entries);
+
+    JSG_TS_OVERRIDE({
+      append(name: string, value: string): void;
+      append(name: string, value: Blob, filename?: string): void;
+
+      set(name: string, value: string): void;
+      set(name: string, value: Blob, filename?: string): void;
+
+      entries(): IterableIterator<[key: string, value: File | string]>;
+      [Symbol.iterator](): IterableIterator<[key: string, value: File | string]>;
+
+      forEach<This = unknown>(callback: (this: This, value: File | string, key: string, parent: FormData) => void, thisArg?: This): void;
+    });
```

Comment:
I think this should also have an override based on [#formdata-parsing-supports-file](https://developers.cloudflare.com/workers/platform/compatibility-dates/#formdata-parsing-supports-file) for `get()` that returns `string | null` instead of `File | string | null` if that flag isn't enabled